### PR TITLE
update theme style doc

### LINKED ISSUE #14094: Document Theme Configuration for Angular UI
Document the required configurations for Angular theme selection (if selected different than the default theme): https://github.com/abpframework/abp/blob/rel-6.0/docs/en/UI/Angular/Theme-Configurations.md

### PR DESCRIPTION
**Pull request by muhammedaltug**

### CODE REVIEW

**Review by mahmut-gundogdu — COMMENTED** [review]
I would describe why we made this break-change. (spoiler:Angular 14+webpack)

File: docs/en/UI/Angular/Theme-Configurations.md [hunk 1]
```diff
@@ -1,3 +1,220 @@
...
+```json
+{
+    "input": "node_modules/bootstrap/dist/css/bootstrap.rtl.min.css",
+    "inject": false,
+    "bundleName": "bootstrap-rtl.min"
+},
+{
+    "input": "node_modules/bootstrap/dist/css/bootstrap.min.css",
+    "inject": true,
+    "bundleName": "bootstrap-ltr.min"
```
Comment (rqx110) [inline docs/en/UI/Angular/Theme-Configurations.md:58]:
```suggestion
    "bundleName": "bootstrap.min"
```

**Review by braim23 — APPROVED** [review]
Sorry i'm late

File: docs/en/UI/Angular/Theme-Configurations.md (hunk 1 above)
Comment (muhammedaltug) [reply docs/en/UI/Angular/Theme-Configurations.md:58]: No, the theme-basic searches for a direction prefix in the bundle name to determine which bundle will load. The [source code](https://github.com/abpframework/abp/blob/ea6cdfd8701421d9af221d2d2480e876d3db64bd/npm/ng-packs/packages/theme-basic/src/lib/handlers/lazy-style.handler.ts#L5)

**Pull request merged**